tests/unit/command/test_checkout.py

```diff
@@ -1,7 +1,6 @@
-import logging
-
 from dvc.cli import parse_args
 from dvc.command.checkout import CmdCheckout, log_changes
+from tests.func.parsing.test_errors import escape_ansi
```

Comment:
`escape_ansi` is temporary for now. I'll move this to test utils later and probably will create a checkout stats ui componet.